README.md

```diff
@@ -3,81 +3,216 @@
+          "version": "3.1.0",
+          "src": "dist/jquery.min.js"
+      },
+      "normalize.css": {
+          "version": "4.2.0",
+          "src": "normalize.css"
+      }
+    }
+  }
+}
+```
+
+Your target folder in your project will look like:
+
+```
+ project
+   |
+   |_ static
+   |   |_ build
+   |        |_ jquery.min.js
+   |        |_ normalize.css
+   |
+
+```
+
+## Full example
+
+```js
+{
+  "name": "frontend-dependencies-test",
+  "version": "1.0.0",
+  "description": "frontend-dependencies test project",
+  "main": "index.js",
+  "author": "Matias Surdi <matias@surdi.net>",
+  "license": "Apache-2.0",
+  "dependencies": { },
+  "devDependencies": {
+    "shelljs": "0.7.4"
+  },
+  "frontendDependencies": {
+    "target": "static/build",
+    "packages": {
+      "jquery": {             // npm package name
+          "version": "3.1.0", // for `npm install`: version, tag or version range
+          "src": "dist/*"     // relative path in package to copy files
+      },
+      "normalize.css": { // copy whole package
+          "version": "4.2.0"
+      },
+      "turbolinks": {
+          // alternative to 'version`: specifie git url, tarball url, tarball file, folder
+          "url": "git://github.com/turbolinks/turbolinks.git",     
+          "src": "{src,LICENSE}", // copy multiple files
+          "target": "static/build/turbo", // specific target path
+          "exact":true // no extra folder with package Name
       }
+    }
+  }
+}
+```
+
+Your target folder in your project will look like:
+
+```
+ project
+   |
+   |_ static
+   |   |_ build
+   |        |_ jquery
+   |        |    |_ core.js
+   |        |    |_ jquery.js
+   |        |    |_ jquery.min.js
+   |        |    |_ ...
+   |        |
+   |        |_ normalize.css
+   |        |    |_ CHANGELOG.md
+   |        |    |_ LICENSE.md
+   |        |    |_ normalize.css
+   |        |    |_ ...
+   |        |
+   |        |_ turbo
+   |              |_ src
+   |              |   |_ turbolinks
+   |              |
+   |              |_ LICENSE
+   |
+   |
+
+```
 
 
+## Installation
 
-### Packages Value Types
+> npm install --save frontend-dependencies
 
+Make it a postinstall script by adding this to your package.json:
+```json
+    "scripts": {
+        "postinstall": "frontend-dependencies"
+    }
+```
 
-#### String
+You can see a complete example [here](https://github.com/msurdi/frontend-dependencies/blob/master/fixtures/package.json)
 
+Run can also run it by hand
 
-Path to a file or directory that is relative to the node_modules directory.
+> /node_modules/.bin/frontend-dependencies
 
+### Packages Options
 
-##### Examples
+#### version
+The npm package name will be taken from the name specified in "frontendDependencies.packages" and the package installed from npm.
+```js
+                         // none: install latest
+    "version": "0.2.4"   // version
+    "version": "beta"    // tag
+    "version": "0.^2.4"  // version range
```

Comment:
Did you mean `^0.2.4` ? First time I see the "^" in this position, but I suppose this could be valid and mean just 0.2.x ? No idea..